tests/test_emails_helpers.py

```diff
@@ -0,0 +1,55 @@
+import json
+
+import pytest
+
+from django.contrib.sites.models import Site
+
+from saleor.order.emails_helpers import (
+    get_order_confirmation_markup, get_organization, get_product_data)
+
+
+def test_get_organization():
+    example_name = 'Saleor Brand Name'
+    site = Site.objects.get_current()
+    site.name = example_name
+    site.save()
+
+    result = get_organization()
+    assert result['name'] == example_name
+
+
+def test_get_product_data_without_image(order_with_lines):
+    """Tested OrderLine Product has no image assigned."""
+    order_line = order_with_lines.get_lines()[0]
+    organization = get_organization()
```

Comment:
I would rather not use in test a function that is not being a subject of the test. Considering it being called also in `test_get_product_data_with_image`, I think we may create fixture for that.